### PR TITLE
Adding Einsteinium 1.3.0 recipe

### PR DESCRIPTION
**Pull request by Perelandra0x309**
New release of Einsteinium 1.3.0

### CODE REVIEW

File: haiku-apps/einsteinium/einsteinium-1.3.0.recipe [hunk 1]
```diff
@@ -0,0 +1,97 @@
+SUMMARY="Monitor applications and system services"
+DESCRIPTION="Einsteinium provides smarter monitoring of applications and \
+system services for Haiku. It can restart applications and system services \
+that were quit or crashed, gather statistics on application usage and \
+provides customizable ranked lists of applications."
+HOMEPAGE="https://sourceforge.net/projects/esforhaiku/"
```
Comment (humdingerb) [inline haiku-apps/einsteinium/einsteinium-1.3.0.recipe]: You may want to point the homepage to https://github.com/Perelandra0x309/einsteinium as well.
You can even create a wiki and simple homepage at github, see e.g. http://humdingerb.github.io/clipdinger/

**Comment by Perelandra0x309** [comment]
Good catch, updated webpage url.  I also removed a leftover user_launch folder I no longer use.

File: haiku-apps/einsteinium/einsteinium-1.3.0.recipe (hunk 1 above)
Comment (Perelandra0x309) [inline haiku-apps/einsteinium/einsteinium-1.3.0.recipe]: Good catch, I've updated the URL and also removed the creation of the user_launch folder I no longer use.

Comment (humdingerb) [inline haiku-apps/einsteinium/einsteinium-1.3.0.recipe]: Thanks!

**Pull request merged**